src/components/layout/AppNavBar.tsx

```diff
@@ -99,13 +99,6 @@ export default function AppNavBar({ navMenuToggle }: AppBarDeckProps) {
                 {keycloak?.authenticated ? <PrivateMenu /> : <PublicMenu />}
                 <Grid item>
                   <LocaleButton />
-                  <IconButton
```

Comment:
I'm a bit sad to see the github icon removed, as it's very well associated with open source.